Dockerfile

```diff
@@ -1,5 +1,14 @@
 FROM deriv/dzil
 ARG HTTP_PROXY
+
+ONBUILD COPY . /opt/app/
+ONBUILD WORKDIR /opt/app/
+ONBUILD RUN /microservice_install.sh
```

Comment:
https://github.com/binary-com/docker-perl/blob/master/prepare-apt-cpan.sh - this is available in `/usr/local/bin` in our standard `deriv/perl` and `deriv/dzil` images.